Directory.Build.props

```diff
@@ -12,6 +12,7 @@
     <Company>Microsoft</Company>
     <Product>Xamarin.Forms</Product>
     <LangVersion>7.3</LangVersion>
+    <ProduceReferenceAssembly Condition="'$(UsingMicrosoftNETSdk)' == 'True'">True</ProduceReferenceAssembly>
```

Comment:
I could not check `TargetFramework` here during MSBuild evaluation, since it was blank. This check sets the value for all netstandard/SDK-style projects.